mautic/mautic
# [UX] Improve cache information from chart dropdown on Dashboard page

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [X]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #13332  



#### Description:
The chart dropdown on the Dashboard page displays technical cache/data information, we can make it better

The idea is to replace things like `data loaded in 3.3 ms from cache` by:
- Showing previously stored data (cache)
- Just retrieved latest data (database)

![image](https://github.com/user-attachments/assets/100a7c24-f7af-4a72-ac2c-b43a6e980930)

![image](https://github.com/user-attachments/assets/bb08c35b-7ebe-4cbc-b1cc-9f92b606f1f5)


to be more user-friendly



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Go to Dashboard
3. Open any chart card dropdown
4. You will see only the cache info
